accepted/0000-registry-url-in-lock-file.md

```diff
@@ -1,102 +0,0 @@
-- Start Date: 2017-09-21
-- RFC PR:
-- Yarn Issue:
-
-# Summary
-
-The yarn.lock file includes information about the registry that it was generated against. It would
-be nice if there was no registry information associated with the lock file
-
-# Motivation
-
-We currently have two internal registries/mirrors. We generate the lock file against one we have in

```

Comment:
I did make an attempt at including this use case, but I didn't understand it well enough to explain it.

Reply:
I think this use case is largely solved by one of those two options:

- Users could simply have a CI build step where they copy `.yarnrc.ci` into `.yarnrc`, or
- Users could call Yarn with the `--use-yarnrc .yarnrc.ci` flag

This would let them define the registries they want to use on CI without too much hassle.